contrib/node/tests/node/node-thrift-js-test/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "node-thrift-js-test",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha --compilers js:babel-register"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.24.1",
+    "babel-preset-latest": "^6.24.1",
+    "babel-register": "^6.24.1",
+    "child-process-promise": "^2.2.1",
+    "mocha": "^3.2.0"
```

Comment:
Should probably update these dependencies using latest versions as we don't want to follow behind with older tests. i.e. mocha is now >= 6.2.0

Reply:
Created an issue for this.